hasjob/templates/macros.html.jinja2

```diff
@@ -9,29 +9,46 @@
     <span class="annotation top-right">{{ post.datetime|shortdate }}</span>
     <span class="headline">{% if is_bgroup %}{{ post.headlineb }}{% else %}{{ post.headline }}{% endif %}</span>
     {%- if show_viewcounts or show_pay %}
-      {%- with post_viewcounts=get_post_viewcounts(post.id) %}
-      <span class="count">
-        {%- if show_viewcounts %}
-          <span title="Listed › Viewed › Opened form › Applied">
-            {{ post_viewcounts['impressions'] }} › {{ post_viewcounts['viewed'] }} › {{ post_viewcounts['opened'] }} › {{ post_viewcounts['applied'] }}
-          </span>
+      {%- set post_viewcounts=get_post_viewcounts(post.id) %}
```

Comment:
Please confirm this works in Jinja 2.9, since the behaviour of `set` has changed. https://github.com/pallets/jinja/issues/641